http-server-jetty/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     implementation libs.jetty.servlet
     implementation(libs.jetty.http2.server)
     implementation(libs.jetty.alpn.server)
+    implementation(mn.micronaut.management)
```

Comment:
this shouldn't be `implementation` we need to figure out a way to make it `compileOnly`